be1-go/network/socket/socket.go

```diff
@@ -94,15 +85,12 @@ func (s *baseSocket) ReadPump(ctx context.Context) {
 			break
 		}
 
-		// TODO: validate message using JSON schema and unmarshal it to generic
-		// message at this stage
```

Comment:
has this been implemented somewhere else?
or should this TODO be moved somewhere else?

Reply:
We do have JSON schema validation in place now: https://github.com/dedis/student_21_pop/blob/3875d8206aeffd7fdd349738e3ac82f20156d2fd/be1-go/hub/base_hub.go#L165